tests/e2e/specs/auth-client-configuration.test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * WordPress dependencies
+ */
+import { activatePlugin, visitAdminPage } from '@wordpress/e2e-test-utils';
+
+describe( 'Providing client configuration', () => {
+
+	beforeEach( async() => {
+		await activatePlugin( 'google-site-kit' );
+		await visitAdminPage( 'admin.php', 'page=googlesitekit-splash' );
+	} );
+
+	it( 'Should have disabled button on load', async() => {
+
+		await page.waitForSelector( '#wizard-step-one-proceed' );
+
+		const proceedButton = await page.$$eval( '#wizard-step-one-proceed', el => el.map( x => x.hasAttribute( 'disabled' ) ) );
+
+		expect( proceedButton.length ).not.toEqual( 0 );
+	} );
+
+
+	it( 'Should have disabled button and display error when input is invalid', async() => {
+
+		await page.waitForSelector( '#client-configuration' );
+		page.click( '#client-configuration' );
+		await page.keyboard.type( 'This is not valid JSON' );
+
+		await page.waitForSelector( '.googlesitekit-error-text' );
+		const errorMessage = await page.$x(
```

Comment:
A simpler method to check for existence of text on a page would be `toMatch` or even better `toMatchElement`:

```js
await expect(page).toMatch( 'Unable to parse client configuration values' );
await expect(page).toMatchElement( '.googlesitekit-error-text', { text: 'Unable to parse client configuration values' } );
```

Might need to add `'expect-puppeteer'` to `setupFilesAfterEnv` to make it work:

https://github.com/google/site-kit-wp/blob/14f71b6899ea211d5751c5dcd123796c62c1f217/tests/e2e/jest.config.js#L5-L7

Reply:
Have this in place now.